benchmarks/configurations/envoy_proxy.yaml

```diff
@@ -0,0 +1,44 @@
+admin:
+  access_log_path: $tmpdir/envoyproxy-admin-access.log
+  profile_path: $tmpdir/envoyproxy.prof
+  address:
+    socket_address: { address: $proxy_ip, port_value: 0 }
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: $proxy_ip
+        port_value: 0
+    # connection_balance_config:
+    #   exact_balance: {}
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          generate_request_id: false
+          codec_type: auto
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: service
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: /
+                route:
+                  cluster: local_service
+          http_filters:
+          - name: envoy.router
+            config:
+              dynamic_stats: false
+  clusters:
+  - name: local_service
+    connect_timeout: 0.25s
+    type: strict_dns
+    lb_policy: round_robin
+    hosts:
+    - socket_address:
+        address: $server_ip
+        port_value: $server_port
```

Comment:
In addition to A/B tests, perhaps we could have A/B/C at some point, where A/B are the before/after Envoy binaries corresponding to some change and C is direct NH client to test server, to demonstrate the available source/sink capacity.

Reply:
I think if we would have a way to annotate tests with group metadata, then we can go all out here in another layer of python scripting, which performs post-processing based on that metadata. E.g. we can treat a group of tests annotated with the same tag as a/b/c/... and crunch the numbers + visualize like that.